android/src/main/java/com/tailscale/ipn/ui/view/ExitNodePicker.kt

```diff
@@ -4,15 +4,151 @@
 
 package com.tailscale.ipn.ui.view
 
-import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.outlined.KeyboardArrowRight
+import androidx.compose.material.icons.outlined.Check
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.ListItem
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
 import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.tailscale.ipn.R
+import com.tailscale.ipn.ui.util.LoadingIndicator
+import com.tailscale.ipn.ui.util.flag
 import com.tailscale.ipn.ui.viewModel.ExitNodePickerViewModel
 
 
+@OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun ExitNodePicker(viewModel: ExitNodePickerViewModel) {
-    Column {
-        Text(text = "Future Home of Picking Exit Nodes")
+fun ExitNodePicker(
+    viewModel: ExitNodePickerViewModel,
+    onNavigateToMullvadCountry: (String) -> Unit,
+) {
+    LoadingIndicator.Wrap {
+        Scaffold(topBar = {
```

Comment:
Scaffold is the idiomatic way to lay out screens in Material. Eventually I think we'll want our own Composable that wraps Scaffold for consistency, but for now I'm using it directly.